### PR TITLE
Dataset refactoring

### PR DESCRIPTION
**Pull request by prashantkul**
Refactored dataset to spit out spinal stenosis as labels.

### CODE REVIEW

**Comment by oa-kwaku** [comment]
taking a look!

File: vision_project/vision_models/imageloader.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        else:
+            train_dataset = train_dataset.batch(constants.BATCH_SIZE)
+            val_dataset = val_dataset.batch(constants.BATCH_SIZE)
+
+        train_dataset = train_dataset.prefetch(tf.data.AUTOTUNE)
+        val_dataset = val_dataset.prefetch(tf.data.AUTOTUNE)
+
+        return train_dataset, val_dataset
+
+# Usage example (commented out):
```
Comment (oa-kwaku) [inline vision_project/vision_models/imageloader.py:270]: ➕ 1 for the usage example

**Review by oa-kwaku — APPROVED** [review]
nice work!
